iroh-rpc/src/stream.rs

```diff
@@ -104,6 +105,8 @@ impl OutStream {
                 })
                 .await
                 .expect("Sender to not be closed.");
+            buf.clear();
+            buf.resize(chunk_size, 0u8);
```

Comment:
I don't think you need to resize on every round

Reply:
Okay so `buf.clear()` makes the `Vec` length 0, so `read_exact` reads in 0 bytes. I need to resize it to the correct amount or else it won't work with `read_exact`

Reply:
You don't actually need to clear it, you can just overwrite it, if the size stays the same